clients/python/test/test_osparc/conftest.py

```diff
@@ -11,3 +12,12 @@ def cfg() -> Iterator[osparc.Configuration]:
         username="askjdg",
         password="asdjbaskjdb",
     )
+
+
+@pytest.fixture
+def enable_dev_mode():
```

Comment:
Q: why not using the pytest monkeypatch for this? is that not exactly the use case?

Reply:
Very good point. Thanks